doc/flame/components.md

```diff
@@ -36,8 +36,8 @@ Every `Component` has a few methods that you can optionally implement, which are
 ```{include} diagrams/component_life_cycle.md
 ```
 
-The `onGameResize` method is called whenever the screen is resized, and once in the beginning when
-the component is added to the game via the `add` method.
+The `onGameResize` method is called whenever the screen is resized, and each time a
```

Comment:
Reading this it implies that the `onGameResize` is called whenever ANY component is added, so my parent `onGameResize` is also called when a child is added to its tree?

I think we should word it more like `The method is called when screen resizes or when the component gets added to the tree`, something in that sense, making it more connected to `the component`

Reply:
done